cli/flox/doc/doc-compose/manifest.toml.md

```diff
@@ -0,0 +1,583 @@
+vars.PGUSER = "myuser"
+vars.PGPASSWORD = "super-secret"
+vars.PGDATABASE = "mydb"
+vars.PGPORT = "9001"
+```
+
+This would define a service called `database` that configures and starts a
+PostgreSQL database.
+
+The full set of options is show below:
+```
+ServiceDescriptor ::= {
+  command    = STRING
+, vars       = null | Map[STRING, STRING]
+, is-daemon  = null | BOOL
+, shutdown   = null | Shutdown
+, systems    = null | [<STRING>, ...]
+}
+
+Shutdown ::= {
+  command = STRING
+}
+```
+
+`command`
+:   The command to run (interpreted by a Bash shell) to start the service. This
+    command can use any environment variables that were set in the `[vars]`
+    section, the `hook.on-activate` script, or the service-specific `vars`
+    table.
+
+`vars`
+:   A table of environment variables to set for the invocation of this specific
+    service. Nothing outside of this service will observe these environment
+    variables.
+
+`is-daemon`
+:   Whether this service spawns a daemon when it starts. Some commands start a
+    background process and then terminate instead of themselves running for an
+    extended period of time. These programs need special handling when it comes
+    time to shut down the services, so you must mark them with the `is-daemon`
+    field. If this field is set to `true` you must also specify the
+    `shutdown.command` field, otherwise the process will continue to run after
+    calling `flox services stop` or after exiting the last activation of the
+    environment.
+
+`shutdown.command`
+:   A command to run to shut down the service instead of delivering the SIGTERM
+    signal to the process. Some programs require special handling to shut down
+    properly e.g. a program that spawns a server process and uses a client to
+    tell the server to shut down. Sending a SIGTERM to a client in that case
+    may not shut down the server. In those cases you may provide a specific
+    shutdown command to run instead of relying on the default behavior of
+    sending a SIGTERM to the service. This field is required if the `is-daemon`
+    field is `true`.
+
+`systems`
+:   An optional list of systems on which to run this service.
+    If omitted, the service is not restricted.
+
+## `[include]`
+
+The `[include]` section of the manifest describes other environments that you'd
+like to merge with the current manifest in order to compose them into a single
+environment.
+
+The list of environments to include is specified by the `include.environments`
+array. The order of the "include descriptors" in this array specifies the
+priority that should be used when merging the manifests. Descriptors later in
+the array take higher priority than those earlier in the array, and manifest
+fields in the composing manifest take the highest priority.
+
+When merging manifests, different sections have different merge semantics.
+
+`[install]`
+: Package descriptors are overwritten entirely by a higher priority manifest.
```

Comment:
(non-blocking) Should we give an example of what this would look like for a package descriptor? Or do you see that detail living somewhere else in the docs?

Reply:
Yes, somehow in describing them I completely forgot to actually show them

Reply:
Fixed